lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingLink.cs

```diff
@@ -84,8 +81,8 @@ public WritableLink(IPEndPoint remoteEndpoint, IInjector injector)
         /// Uses the already connected TcpClient.
         /// </summary>
         /// <param name="client">The already connected client</param>
-        /// <param name="injector">The injector to pass arguments to incoming messages</param>
-        public WritableLink(TcpClient client, IInjector injector)
+        /// <param name="streamingCodec">Streaming codec</param>
+        public StreamingLink(TcpClient client, IStreamingCodec<T> streamingCodec)
```

Comment:
There is a problem doing that....I looked in google and a constructor cannot be called from inside the body of another constructor. The commeans base() or this() works only before the body of constructor. And I need the Client to be created in another constructor first since initializing other field use this client. Another option I considered was to create a common void SetUp() method. But for that I have to remove readonly keyword from the private fields in the class. Which one is more desirable: letting the two constructors be like this or removing readonly keyword.